grpc-proto/proto/schema.proto

```diff
@@ -49,12 +49,13 @@ message CqlTable {
   repeated ColumnSpec clusteringKeyColumns = 4;
   repeated ColumnSpec staticColumns = 5;
   repeated ColumnSpec columns = 6;
-  map <string, ColumnOrderBy> clusteringKeyColumnSortOrders = 7;
+  map <string, ColumnOrderBy> clusteringOrders = 7;
 }
 
 message CqlTableCreate {
-  CqlTable table = 1;
-  bool ifNotExists = 2;
+  string keyspaceName = 1;
```

Comment:
This is backwards-incompatible change (changing of field ids) and we probably do not/cannot make it.
So maybe add `keyspaceName` as `3` (the usual way to add things)?

Reply:
This is a proto definition that has only ever existed on this branch and has never been released. I don't think we need to be that strict in this case.

Reply:
We just added that message in v2, I don't think we need to preserve backward compatibility yet.

Reply:
Ah... Ok sorry, did not realize it was change to what we added. That's fine then.